Mlem/Views/Shared/Links/Community Link View.swift

```diff
@@ -48,17 +53,19 @@ struct CommunityLinkView: View {
                             .scaledToFit()
                             .frame(width: AppConstants.largeAvatarSize, height: AppConstants.largeAvatarSize)
                     } else {
-                        Image("Default Community")
+                        Image(systemName: "building.2.crop.circle.fill")
                             .resizable()
                             .scaledToFit()
-                            .frame(width: AppConstants.defaultAvatarSize, height: AppConstants.defaultAvatarSize)
+                            .frame(width: AppConstants.largeAvatarSize, height: AppConstants.largeAvatarSize)
+                            .foregroundColor(.secondary)
                     }
                 }
             } else {
-                Image("Default Community")
+                Image(systemName: "building.2.crop.circle.fill")
```

Comment:
At some point we should make a class that keeps our SFSymbols references in one place, so if we change context usage, it updates everywhere.

Reply:
I'll slate it for 1.1